haskell/yesod/README.md

```diff
@@ -0,0 +1,118 @@
+# yesod docker image
+
+Use this image as the base image for your yesod based applications.
+
+## Usage
+
+The simplest usage is to create a `Dockerfile` for your project that looks like
+this:
+```
+FROM thoughtbot/yesod
+
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY *.cabal ./
+RUN cabal install --dependencies-only -j4 --enable-tests
+```
+
+## Setup Your Yesod Project
+
+First, create a `Dockerfile` for your project. The one shown above is a good
+starting place.
+
+Test that your container builds before moving on:
```

Comment:
If they test their container at this point and then switch to docker-compose, they'll need to reinstall the dependencies, which takes like 20 minutes. Is there a reason to test here?